licensing-activation/licensed-users-count.md

```diff
@@ -18,14 +19,27 @@ Free Microsoft licenses assigned - for example, Microsoft Flow Free - are also c
 {% endhint %}
 
 The number of licensed Users within Syskit Point can be found by going to **Settings (1) > General (2) > Point License (3) > Number of Users M/N (4)**.
+* You can also see a report with all your licenses by clicking the **View Point Licenses button (5)**. 
 
 ![Number of Licensed Users](../.gitbook/assets/activation-licensed-users-point.png)
 
  * **M** is the **number of licensed users in the tenant** 
  * **N** is the **number of Paid users that are included in the Syskit Point license**
 
-**The exact number and type of licenses** that are counted toward your Syskit Point subscription can be found on the **Licensed Users tile**. 
-* Click **Users (1)** on the Home page
+**Clicking the View Point Licenses button** opens the Point Licenses report screen with a list of all licenses that are assigned to users in Syskit Point. 
+
+On that screen, you will find the following information:
+  * **Display Name (1)** of the user
+  * **Licenses (2)** that are assigned to the user
+  * **Username (3)** shows the email of the user
+  * **Department (4)** that the user belongs to 
+
+You can also use the **column chooser (5)** to add additional columns to the report. 
+
+![View Point Licenses - Report](../.gitbook/assets/activation-licensed-users_licenses-report.png)
+
+**The exact number and type of licenses** that are counted toward your Syskit Point subscription can be found on the **Licensed Users screen**. 
```

Comment:
ovo mislim da nije točno
tu će se prikazati useri sa bilo kojim tipom licenci, bile one free ili ne